frontend/src/App.js

```diff
@@ -17,7 +29,7 @@ function App() {
         >
           Learn React
         </a>
-      </header>
+      </header> */}
```

Comment:
If you comment out code but keep it within your app I would suggest to add a short comment above that states why this code is left within your app. E.g.: 
`// todo task/#20 fix bug xy <link-to-ticket> `
or
`// !! Kept due to reference reasons - Delete if ...`